tests/unit/test_simple_neuromodulation.py

```diff
@@ -0,0 +1,159 @@
+"""Test neuromodulations using the toy ringtest data as such:
+neuromodulatory projection: virtual_neurons->RingB, 0->0, 1->0
+The post synatic cell RingB gid 0 has 2 synapses:
+    RingA->RingB : 0->0
+    RingB->RingB : 1->0
+The neuromodulatory spikes are injected via replay from virtual neuron gid 1
+Therefore, the neuromodulator applies netcon to the synapse from connection RingB->RingB : 1->0
+"""
```

Comment:
if I understand correctly it is neurodamus that assigns a certain synapse to a certain neuromodulation based on "proximity". We are not really testing that part here as we just assume that it gets the correct synapse and we roll with that. 

Maybe you could add a test to verify that this assignment works even for the edge cases

Reply:
We don't keep that `found` synapse obj to the end, rather to store the `netcon` targeting that synapse if a replay is injected. Let me try what I can do more.

Reply:
A new test is added. Please check